src/modules/docker.ts

```diff
@@ -143,9 +144,20 @@ export default class Docker {
 				container.image = image[0];
 				container.version = image[1];
 				container.ports = data.services[service].hasOwnProperty('ports') ? data.services[service].ports : [];
+
+				// Only add information to output if the service has an image defined else its simple overrides.
+				containers.push(container);
+			}
+
+			// Check if this is a build.
+			if (data.services[service].hasOwnProperty('build')) {
+				container.image = data.services[service]['build'];
+				container.version = 'unknown';
```

Comment:
This could be either `container.version = 'build';` or we could an extra key: 
`container.source = 'build';` / `container.source = 'hub';`

Reply:
Fixed